conda_libmamba_solver/solver.py

```diff
@@ -171,21 +172,23 @@ def solve_final_state(
         else:
             IndexHelper = LibMambaIndexHelper
 
-        if os.getenv("CONDA_LIBMAMBA_SOLVER_NO_CHANNELS_FROM_INSTALLED") or (
-            getattr(context, "_argparse_args", None) or {}
-        ).get("override_channels"):
-            # see https://github.com/conda/conda-libmamba-solver/issues/108
-            channels_from_installed = ()
-        else:
-            channels_from_installed = in_state.channels_from_installed()
-
-        all_channels = (
+        all_channels = [
             *conda_bld_channels,
             *self.channels,
             *in_state.channels_from_specs(),
-            *channels_from_installed,
-            *in_state.maybe_free_channel(),
-        )
+        ]
+        override = (getattr(context, "_argparse_args", None) or {}).get("override_channels")
```

Comment:
I wonder if `conda` should have a `context` flag for `override_channels`. This is a bit clunky and, unless I am missing a good reason, unnecessary.

Reply:
agreed

I suspect the reasoning was because it would mean users could set it in their condarc and override channel settings set by sys admins